ci-operator/step-registry/ipi/conf/aws/windows-machineset/ipi-conf-aws-windows-machineset-ref.yaml

```diff
@@ -1,6 +1,9 @@
 ref:
   as: ipi-conf-aws-windows-machineset
-  from: tests
+  from_image:
+    namespace: ocp
+    name: "4.10"
```

Comment:
Why is this pinned to `4.10`?

Reply:
I may not have a concrete answer for that. I followed the pattern used in the [ipi-conf-aws-ref.yaml](https://github.com/openshift/release/blob/124097c969f5bb91e54c6b352569eb914c16f79e/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L5) step and set `4.10` to correlate WMCO 5.0.z  with a supported Windows Server version, like Windows Server 2019 used in the [script](https://github.com/openshift/release/blob/5ef0d894bd54f9ed7b7cf29fb4b7407a2191d3d6/ci-operator/step-registry/ipi/conf/aws/windows-machineset/ipi-conf-aws-windows-machineset-commands.sh#L19) that creates the machineSet.

I need an image that:
- is available in CI registry
- contains the `aws`  tool
- contains the `oc`  tool
- contains the `jq` tool

Reply:
Won't the WMCO and OCP correlation change based on what branch this test is being run against? Or is it okay to have the version skew?

Reply:
> Won't the WMCO and OCP correlation change based on what branch this test is being run against?

Yes, they change and this clould be tricky. Given WMCO is not part of the OCP payload, there may not be a WMCO version available for all OCP releases, especially the latest ones.

> is it okay to have the version skew?

Yes, it's fine. This image will be used only to run the script that creates the machineSet.

Reply:
>Given WMCO is not part of the OCP payload, there may not be a WMCO version available for all OCP releases, especially the latest ones.

Eventually there will be a WMCO version for every OCP version. 

>Yes, it's fine. This image will be used only to run the script that creates the machineSet.

Given this, I agree it does not matter.